### ISSUE TITLE
A little problem about font load

### ISSUE THREAD

**Issue by bigshans** [Daniele-rolli/Beaver-Notes#70]
I always get "403" error when fonts are loading, which causes the app startup slowly.

![图片](https://github.com/Daniele-rolli/Beaver-Notes/assets/26884666/eab851d1-f8dd-405b-8b87-59f6c352ed56)

I temporary removed this code from `tailwind.css` and rebuild the app. And now my app is back to normal.

```css
@import url('https://fonts.googleapis.com/css2?family=Arimo:ital,wght@0,400;0,500;1,400&family=EB+Garamond:ital,wght@0,400;0,500;1,400&family=Ubuntu:ital,wght@0,400;0,500;1,400&display=swap');@import url('https://fonts.cdnfonts.com/css/open-dyslexic');
@import url('https://fonts.cdnfonts.com/css/helvetica-neue-55');
@import url('https://fonts.cdnfonts.com/css/avenir');
```

I'm not very sure it really need to loading font configuration from remote.

**Comment by Daniele-rolli** [comment]
I'll look into it. At first glance, it might be an issue with the GA. However, you make a compelling case when you say that the font configuration doesn't really need to be remote. We already do this for the code block, so we could do the same here.

**Comment by Daniele-rolli** [comment]
Okay, I've addressed this issue in commit cb8ace0, so now the fonts are integrated directly into the app. Furthermore, I've implemented the same method in Beaver Pocket to prevent this issue from arising again, and more importantly, to ensure that fonts work offline as well.
